.keycloak/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# This Dockerfile will build an arm64 Keycloak image that can be used in the same way
```

Comment:
tested fine locally, hope we can remove it soon in near future 🤞

Reply:
I also hope so 🙏 . Until then some people might be pretty happy to have a one-liner to build the image